hadoop-yarn-project/pom.xml

```diff
@@ -90,6 +91,56 @@
       <artifactId>hadoop-yarn-applications-catalog-webapp</artifactId>
       <type>war</type>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
shouldn't all these be scoped as provided?

Reply:
Yes, the scope here should be defined as 'provided'.

Reply:
@steveloughran  all these dependencies already changed to provided   scope.